### PR TITLE
Updated year + Suppleties

### PR DESCRIPTION
**Pull request by AlexanderErbasu**
Updated time-slider
2020 -> 2021

### CODE REVIEW

File: src/components/TimeSliderSettings.vue [hunk 1]
```diff
@@ -68,7 +68,7 @@
                 ref="endDatePicker"
                 v-model="endDate"
                 :min="startDate"
-                max="2020"
+                max="2021"
```
Comment (CindyvdVries) [inline src/components/TimeSliderSettings.vue:71]: not per se for this PR - but we should have one central place to set this 'year' (env or conf), now coming back in different places thoughout the code

File: src/components/MapComponent.vue [hunk 2]
```diff
@@ -153,56 +202,74 @@ export default {
...
               propsData: {
-                tableHeaders: this.tableHeaders,
-                tableItems: this.tableItems
+                tableHeaders: tableSelectionHeaders,
+                tableItems: this.tableSelectionItems
               }
-            }).$mount('#vue-popup-content')
+            }).$mount('#vue-popup-selection-content')
+          }
+          else if (this.nourishmentsArea.length==1){
```
Comment (CindyvdVries) [inline src/components/MapComponent.vue:270]: javascript ===

File: src/components/MapComponent.vue [hunk 3]
```diff
@@ -130,6 +145,40 @@ export default {
...
+
+      new this.dataTable({
+        propsData: {
+          tableHeaders: this.tableHeaders,
+          tableItems: this.tableItems
+        }
+      }).$mount('#vue-popup-content')
+    },
+
+    selectSuppletie(number_nourishment) {
```
Comment (CindyvdVries) [inline src/components/MapComponent.vue:178]: english/dutch naming

use camelCase for javascript params

File: src/components/MapComponent.vue [hunk 4]
```diff
@@ -130,6 +145,40 @@ export default {
...
       })
     },
+    createMultipleSelectPopup() {
+      this.selectionPopup = new mapboxgl.Popup({
+        closeButton: true,
+        closeOnClick: true
+      })
+    },
+
+    writePopUp(elem) {
```
Comment (CindyvdVries) [inline src/components/MapComponent.vue:155]: not very descriptive. Please add comments (try to do that for each new function at least and where code needs explaining)

File: src/components/DataSelectionTable.vue [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      type: Array
+    },
+    tableItems: {
+      required: true,
+      type: Array
+    }
+  },
+  methods: {
+    buttonPressed(num) {
+      bus.$emit('suppletiesRowSelected', num)
```
Comment (CindyvdVries) [inline src/components/DataSelectionTable.vue:35]: dutch/english naming

**Review by CindyvdVries — APPROVED** [review]
looks good, 

try using the fix lint option

File: src/components/TimeSlider.vue [hunk 6]
```diff
@@ -44,7 +44,7 @@ export default {
         moment()
           .subtract(10, 'years')
           .startOf('year'),
-        moment().startOf('year')
+        moment().startOf('year').add(1, 'year')
```
[the remark this answers is not in the document]
Comment (AlexanderErbasu) [reply src/components/TimeSlider.vue:47]: Yeah, I thought about this as well, but I didn't have the time to fix that as well ^^"

**Pull request merged**